the-experts-BE/database/db_build.sql

```diff
@@ -1,23 +1,56 @@
 BEGIN;
-    DROP TABLE IF EXISTS Contractor;
 
-    CREATE TABLE
-    IF NOT EXISTS Contractor
+DROP TABLE IF EXISTS Contractor,
+Users;
+
+CREATE TABLE IF NOT EXISTS Contractor (
+    id SERIAL PRIMARY KEY,
+    cont_name VARCHAR (50) NOT NULL,
+    job TEXT NOT NULL,
+    rating double precision DEFAULT 0
+);
+
+CREATE TABLE IF NOT EXISTS Users (
+    id SERIAL PRIMARY KEY,
+    first_name VARCHAR (50) NOT NULL,
+    last_name VARCHAR (50) NOT NULL,
+    email TEXT NOT NULL,
+    user_password TEXT NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS Reviews (
+    id SERIAL,
+    user_id int NOT NULL,
+    contractor_id int NOT NULL,
+    reviewBody TEXT,
+    isliked BOOLEAN,
+    FOREIGN KEY (contractor_id) REFERENCES Contractor (id),
+    FOREIGN KEY (user_id) REFERENCES Users (id)
+);
+
+INSERT INTO
+    Users (first_name, last_name, email, user_password)
+VALUES
     (
-        id SERIAL PRIMARY KEY,
-        cont_name VARCHAR
-    (50) NOT NULL,
-        job TEXT NOT NULL,
-        rating double precision	 DEFAULT 0
+        'ebraheem',
+        'abbas',
+        'ebraheemabbas51@gmail.com',
+        'password'
```

Comment:
Same issue with hashed password here, if it's being hashed then you should put a hashed password in here rather than `'password'`